.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  push:
+  pull_request:
```

Comment:
you're not specifying which branch you want to run against

Reply:
Changed to run generally on all push / pull request events